_delphi_utils_python/delphi_utils/geomap.py

```diff
@@ -6,15 +6,12 @@
 TODO:
 - use a caching utility to store the crossfiles
   see: https://github.com/cmu-delphi/covidcast-indicators/issues/282
-- remove deprecated functions once integration into JHU and Quidel is refactored
-  see: https://github.com/cmu-delphi/covidcast-indicators/issues/283
 """
 # pylint: disable=too-many-lines
 from os.path import join
-import warnings
-import pkg_resources
 
 import pandas as pd
+import pkg_resources
```

Comment:
Why is this needed?

Reply:
it just got auto-reordered by pycharm when I was cleaning up imports, used to be line 15.